flake.nix

```
@@ -77,6 +77,15 @@
         description = "A simple Go package";
       };
 
+      haskell.nix = {
+        path = ./go-hello;
```

This is bringing in the wrong template

```suggestion
        path = ./haskell.nix;
```